### PR TITLE
Feature/user model

### PR DESCRIPTION
**Pull request by poom201211**
Please check my code

### CODE REVIEW

File: tests/test_user.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from app.models import Event,User
```
Comment (pknn1) [inline tests/test_user.py]: Remove unused import

File: tests/test_event.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import datetime
-from app.models import Event
+from app.models import Event,User
```
Comment (pknn1) [inline tests/test_event.py]: Remove unused import

File: app/models.py [hunk 3]
```diff
@@ -1,7 +1,23 @@
...
 
 
+class User(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    username = db.Column(db.String(64), index=True, unique=True)
+    email = db.Column(db.String(120), index=True, unique=True)
+    password_hash = db.Column(db.String(128))
+
+    def __repr__(self):
+        return '<User {}>'.format(self.username)
```
Comment (pknn1) [inline app/models.py]: Use string literal instead of `.format()`

File: app/models.py [hunk 4]
```diff
@@ -10,7 +10,7 @@ class User(db.Model):
     password_hash = db.Column(db.String(128))
 
     def __repr__(self):
-        return '<User {}>'.format(self.username)
+        return '<User {}>'.db.String(self.username)
```
Comment (pknn1) [inline app/models.py]: Use `return f'<User {self.username}>'`

File: tests/test_user.py (hunk 1 above)
Comment (pknn1) [reply tests/test_user.py]: Here's too.

**Comment by poom201211** [comment]
alright

**Pull request merged**